modules/administration/pages/users.adoc

```diff
@@ -153,3 +154,27 @@ Where a translation is not available, the {webui} will default to English (``en_
 ====
 
 endif::[]
+
+
+
+=== User Default Interface Theme
+
+By default, the {productname} {webui} uses the theme appropriate to the product you have installed.
+You can change the theme to reflect the {uyuni} or {susemgr} colors.
+The {susemgr} theme also has a dark option available.
+
+After a user has been created, you can change the default theme in the ``rhn.conf`` configuration file.
```

Comment:
this does not seem to depend on the created user.  it seems to be a system-wide setting.